extensions/wikia/CuratedContent/CuratedContentValidator.class.php

```diff
@@ -160,7 +160,15 @@ public function validateItem( $item ) {
 			$this->error( $item, self::ERR_ARTICLE_NOT_FOUND );
 		}
 
-		$this->titles[] = $item['title'];
+		$this->addLabelForValidation( $item['label'], $item['title'] );
+	}
+
+	private function addLabelForValidation( $label, $title ) {
+		if ( array_key_exists( $label, $this->existingLabels ) ) {
+			$this->existingLabels[$label][] = $title;
+		}  else {
+			$this->existingLabels[$label] = [ $title ];
```

Comment:
please simplify.
This $this->existingLabels[$label][] = $title; is enough no ifs ;)